raiden-ts/src/config.ts

```diff
@@ -112,15 +112,20 @@ export function makeDefaultConfig(
   { network }: { network: Network },
   overwrites?: PartialRaidenConfig,
 ): RaidenConfig {
+  const networkName = getNetworkName(network);
+  const matrixServerInfos =
+    networkName === `mainnet`
```

Comment:
@andrevmatos Should we compare using this name, or the `chain_id`?

Reply:
On `raiden.ts:1126`, we use `network.chainId !== 1` to check for mainnet.. I think it'd be better than using this string which is just intended to be a _human-readable_ name of it, although it works.